lib/presentation/menu/widgets/build_information_tile.dart

```diff
@@ -15,7 +15,7 @@ class BuildInformationTile extends StatelessWidget {
         const SizedBox(height: 50),
         const SizedBox(
           width: 56,
-          child: Image(image: AssetImage('assets/images/build_info.png')),
```

Comment:
If we're not using `build_info.png` anywhere else, we can delete it from the assets folder.